flink-ai-flow/lib/airflow/airflow/contrib/jobs/event_based_scheduler_job.py

```diff
@@ -344,8 +344,15 @@ def _find_dagruns_by_event(self, event, session) -> Optional[List[DagRun]]:
             dep: DagEventDependencies = DagEventDependencies.from_json(dag.event_relationships)
             if dep.is_affect(event_key):
                 context_extractor: ContextExtractor = dag.context_extractor
-                event_context: EventContext = context_extractor.extract_context(event)
-                affect_dags[dag.dag_id] = event_context
+                try:
+                    event_context: EventContext = context_extractor.extract_context(event)
+                except Exception as e:
+                    self.log.error(
+                        "Failed to call context extractor, dag {} skips event {}".format(dag.dag_id, event),
+                        exc_info=e)
+                    continue
+                if event_context is not None:
+                    affect_dags[dag.dag_id] = event_context
```

Comment:
It would be better if we can add some log here when `event_context` is None, which will tell users that they write wrong context_extractor.